snapshot/lib/assets/SnapshotHelper.swift

```diff
@@ -118,12 +120,30 @@ open class Snapshot: NSObject {
         }
     }
 
-    class func pathPrefix() -> NSString? {
-        if let path = ProcessInfo().environment["SIMULATOR_HOST_HOME"] as NSString? {
-            return path.appendingPathComponent("Library/Caches/tools.fastlane") as NSString?
-        }
-        print("Couldn't find Snapshot configuration files at ~/Library/Caches/tools.fastlane")
-        return nil
+    class func pathPrefix() -> URL? {
+        let homeDir: URL
+        //on OSX config is stored in /Users/<username>/Library
+        //and on iOS/tvOS/WatchOS it's in simulator's home dir
+        #if os(OSX)
+            guard let user = ProcessInfo().environment["USER"] else {
+                print("Couldn't find Snapshot configuration files - can't detect current user ")
+                return nil
+            }
+        
+            guard let usersDir =  FileManager.default.urls(for:.userDirectory, in:.localDomainMask).first else {
+                print("Couldn't find Snapshot configuration files - can't detect `Users` dir")
+                return nil
+            }
+            
+            homeDir = usersDir.appendingPathComponent(user)
+        #else
+            guard let homeDirUrl = URL(string: ProcessInfo().environment["SIMULATOR_HOST_HOME"]!) else {
```

Comment:
Just noticed this `!`... could we add in a check here so that we make sure that `ProcessInfo().environment["SIMULATOR_HOST_HOME"]` exists as a String before passing it in to `URL(string:)`?